### PR TITLE
Adding languages- French and Portuguese

### PR DESCRIPTION
**Pull request by simo9900**
**Closes: #Issue**

### Checklist

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Comment by simo9900** [comment]
Adding languages- French and Portuguese

File: lib/src/localization.dart [hunk 1]
```diff
@@ -79,22 +85,22 @@ class TimetableLocalizationsDelegate
...
           'The timetable library uses Localizations to generate messages, '
-          'labels, and abbreviations.',
+              'labels, and abbreviations.',
         ),
         ErrorHint(
           'To introduce a TimetableLocalization, add a '
-          'TimetableLocalizationsDelegate() to your '
-          "Material-/Cupertino-/WidgetApp's localizationsDelegates.",
+              'TimetableLocalizationsDelegate() to your '
+              "Material-/Cupertino-/WidgetApp's localizationsDelegates.",
```
Comment (JonasWanke) [inline lib/src/localization.dart:103]: Could you please revert these formatting changes?

File: README.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Some date/time-related parameters also have special suffixes:
 * `timeOfDay`: A `Duration` between zero and 24 hours.
 * `dayOfWeek`: An `int` between one and seven ([`DateTime.monday`](https://api.flutter.dev/flutter/dart-core/DateTime/monday-constant.html) through [`DateTime.sunday`](https://api.flutter.dev/flutter/dart-core/DateTime/sunday-constant.html)).
 
-Timetable currently offers localizations for Chinese, English, German, Italian, Japanese, and Spanish.
+Timetable currently offers localizations for Chinese, English, French, German, Italian, Japanese, Portuguese and Spanish.
```
Comment (JonasWanke) [inline README.md:64]:
```suggestion
Timetable currently offers localizations for Chinese, English, French, German, Italian, Japanese, Portuguese, and Spanish.
```

**Review by JonasWanke — CHANGES_REQUESTED** [review]
Thanks for the contribution! I've added two minor comments; when these are fixed, I'm happy to merge this PR

**Review by JonasWanke — APPROVED** [review]

**Comment by JonasWanke** [comment]
This is now published as part of [v1.0.0-alpha.5](https://pub.dev/packages/timetable/versions/1.0.0-alpha.5/changelog#100-alpha5--2021-08-07). Thanks for contributing!

**Pull request merged**